lib/ui/painting.dart

```diff
@@ -1839,6 +1839,15 @@ class Image {
     return stacks;
   }
 
+  /// Returns the approximate byte size of the allocation of this image.
+  ///
+  /// This takes into account details such as mip-mapping. The allocation
+  /// is usually resident in the device memory.
```

Comment:
We should document this as being accurate for the storage format and mipmapping of the image, but not document where the allocation is resident - for most images, they're actually CPU resident.

Reply:
done

Reply:
Btw this looks interesting: DlImage says "Represents an image whose allocation is (usually) resident on device memory", while the CanvasImage (as a simple wrapper of DlImage) is on CPU.